docs/source/API/core/atomics/atomic_fetch_op.rst

```diff
@@ -0,0 +1,104 @@
+``atomic_fetch_[op]``
+=====================
+
+.. role:: cppkokkos(code)
+   :language: cppkokkos
+
+Header File: <Kokkos_Core.hpp>
+
+Usage
+-----
+
+.. code-block:: cpp
+
+   old_value =  atomic_fetch_[op](ptr_to_value, update_value);
+
+Atomically updates the variable at the address given by ``ptr_to_value`` with ``update_value``
```

Comment:
Is it correct that this `atomic_fetch_add` (for instance) returns the previous value at that address vs. the previous values.